core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooQueueLock.java

```diff
@@ -42,6 +42,19 @@
   private String path;
   private boolean ephemeral;
 
+  public static class FateLockPath {
+    private final String path;
+
+    public FateLockPath(String path) {
+      this.path = path;
+    }
+
+    @Override
+    public String toString() {
+      return this.path;
+    }
+  }
+
   public ZooQueueLock(ZooReaderWriter zrw, String path, boolean ephemeral) {
```

Comment:
Can this be converted to use FateLockPath?

Reply:
Yes, I overlooked this. Good catch, I will add this in my next commit.

Reply:
Made this change in my latest commit